benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for setting up an integration of an OSS-Fuzz benchmark. The script
+will create oss-fuzz.yaml as well as copy the files from OSS-Fuzz to build the
+benchmark."""
+import argparse
+import datetime
+from distutils import dir_util
+import os
+import sys
+
+from common import utils
+
+# pytype: disable=import-error
+# pylint: disable=import-error,wrong-import-position,ungrouped-imports,too-many-arguments
+
+# TODO(metzman): Don't rely on OSS-Fuzz code. We don't want to depend on it
+# because it can easily break use. Especially becuase:
+# 1. We use private methods
+# 2. The OSS-Fuzz code depends on absolute imports.
+# 3. The OSS-Fuzz code assumes it is run from the OSS-Fuzz directory and can
+# accdidentaly break our repo.
+OSS_FUZZ_REPO_PATH = os.path.join(utils.ROOT_DIR, 'third_party', 'oss-fuzz',
+                                  'infra')
+sys.path.append(OSS_FUZZ_REPO_PATH)
+
+import bisector
+import build_specified_commit
+import helper
+import repo_manager
+
+from common import logs
+from common import yaml_utils
+
+
+def copy_dir_contents(src_dir, dst_dir):
+    """Copy the contents of |src_dir| into |dst_dir|."""
+    return dir_util.copy_tree(src_dir, dst_dir)
```

Comment:
why this instead of shutil.copytree ?

Reply:
Uses shutil.copytree now.

Reply:
I'm pretty sure copytree didn't work and that is why I used dir_util.copy_tree.
I can't exactly remember why but I think it's because copy tree will copy the directory including the name, whereas dir_util.copytree copies the *contents*

Reply:
Right, that works for a test entry but not if it exists. I'll change that.